### DISCUSSION TITLE
amt random_steps function

### DISCUSSION THREAD

**Discussion by KirstenFuller** [jmsigner/amt#98] [General]
Hello all! I am trying to generate directed and correlated random steps to pair with observed steps in my dataset of eagle movements. I think the random_steps function in the amt package accomplishes the paired random steps, but can anyone tell me if these are directed as well? Thank you!

**Comment by jmsigner** [comment]
Hi, 

the `random_steps()`function will fit a distribution to turn angles and step lengths and sample from this distribution by default. Alternatively, you can use the `ta_distr` argument and pass a "very" directed von Mises distribution. 

HTH
